samples/drivers/espi/src/main.c

```diff
@@ -1212,62 +1203,52 @@ int espi_test(void)
 	k_sleep(K_SECONDS(1));
 
 #if DT_NODE_HAS_STATUS(BRD_PWR_NODE, okay)
-	pwrgd_dev = device_get_binding(BRD_PWR_PWRGD);
-	if (!pwrgd_dev) {
-		LOG_WRN("%s not found", BRD_PWR_PWRGD);
-		return -1;
+	if (!device_is_ready(pwrgd_gpio.port)) {
+		LOG_ERR("%s: device not ready.", pwrgd_gpio.port->name);
+		return -ENODEV;
 	}
-
-	rsm_dev = device_get_binding(BRD_PWR_RSMRST);
-	if (!rsm_dev) {
-		LOG_WRN("%s not found", BRD_PWR_RSMRST);
-		return -1;
+	if (!device_is_ready(rsm_gpio.port)) {
+		LOG_ERR("%s: device not ready.", rsm_gpio.port->name);
+		return -ENODEV;
 	}
-
 #endif
-	espi_dev = device_get_binding(ESPI_DEV);
-	if (!espi_dev) {
-		LOG_WRN("Fail to find %s", ESPI_DEV);
-		return -1;
+	if (!device_is_ready(espi_dev)) {
+		LOG_ERR("%s: device not ready.", espi_dev->name);
+		return -ENODEV;
 	}
 
 #ifdef CONFIG_ESPI_SAF
-	qspi_dev = device_get_binding(SPI_DEV);
-	if (!qspi_dev) {
-		LOG_WRN("Fail to find %s", SPI_DEV);
-		return -1;
+	qspi_dev = DEVICE_DT_GET(SPI_NODE);
+	if (!device_is_ready(qspi_dev)) {
+		LOG_ERR("%s: device not ready.", qspi_dev->name);
+		return -ENODEV;
 	}
 
-	espi_saf_dev = device_get_binding(ESPI_SAF_DEV);
-	if (!espi_saf_dev) {
-		LOG_WRN("Fail to find %s", ESPI_SAF_DEV);
-		return -1;
+	espi_saf_dev = DEVICE_DT_GET(ESPI_SAF_NODE);
+	if (!device_is_ready(espi_saf_dev)) {
+		LOG_ERR("%s: device not ready.", espi_saf_dev->name);
+		return -ENODEV;
 	}
 #endif
 
 	LOG_INF("Hello eSPI test %s", CONFIG_BOARD);
 
 #if DT_NODE_HAS_STATUS(BRD_PWR_NODE, okay)
-	ret = gpio_pin_configure(pwrgd_dev, BRD_PWR_PWRGD_PIN,
-				 GPIO_INPUT | GPIO_ACTIVE_HIGH);
+	ret = gpio_pin_configure_dt(&pwrgd_gpio, GPIO_INPUT);
```

Comment:
is `GPIO_ACTIVE_HIGH` reflected in DT now?

Reply:
Yes, it was already in the DTS.

Reply:
Yes, it was already in the DTS.